x-pack/plugins/canvas/public/components/datasource/__stories__/datasource_component.stories.tsx

```diff
@@ -90,5 +91,6 @@ storiesOf('components/datasource/DatasourceComponent', module)
       setPreviewing={action('setPreviewing')}
       isInvalid={false}
       setInvalid={action('setInvalid')}
+      renderError={action('renderError')}
```

Comment:
Does this belong?

Reply:
Yes, those changes are required for successfully passing CI pipeline.